src/Uno.UI.RemoteControl/RemoteControlClient.cs

```diff
@@ -31,6 +31,18 @@ public ReceivedFrameEventArgs(HotReload.Messages.Frame frame)
 		public HotReload.Messages.Frame Frame { get; set; }
 	}
 
+	public class ClientEventEventArgs : EventArgs
+	{
+		public ClientEventEventArgs(string eventName, string eventDetails)
+		{
+			EventName = eventName;
+			EventDetails = eventDetails;
+		}
+
+		public string EventName { get; }
+		public string EventDetails { get; }
+	}
+
```

Comment:
Let's move this class and the one above to separate classes. It could probably be records as well.

Reply:
moved